src/assets/styles/base/_typography.scss

```diff
@@ -1,38 +1,21 @@
-body {
-  color: $body-color;
-  font-family: $font-family-sans-serif-light;
-  font-size: .8em;
-  -webkit-font-smoothing: subpixel-antialiased;
-  letter-spacing: $letter-spacing-base;
-  line-height: $line-height;
-
-  @media screen and (min-width: map-get($bp, "md")) {
-    font-size: .9em;
-  }
-
-  @media screen and (min-width: map-get($bp, "lg")) {
-    font-size: 1em;
-  }
-}
-
 h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  color: $heading-color;
-  font-family: $heading-font-family;
-  font-weight: $font-weight-base;
-  line-height: $heading-line-height;
+  color: $color-grey-dark;
+  font-family: $font-family-sans-serif-medium;
+  font-weight: 400;
```

Comment:
Why have we lost our heading variables on all of these?

Reply:
Need a bit of clarity on this one if that's okay?